src/autoscaler/db/sqldb/appmetric_sqldb_test.go

```diff
@@ -1,6 +1,7 @@
 package sqldb_test
 
 import (
+	context2 "context"
```

Comment:
why importing with an alias?

Reply:
Thanks, that's an artifact of IDEA's refactor method signature.